.idea/JobFunnel.iml

```diff
@@ -2,14 +2,13 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Pipenv (JobFunnel)" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.6" jdkType="Python SDK" />
```

Comment:
is this just the name of the project in pycharm?

Reply:
Yeah, sorry about that. Was trying out pycharm and apparently it added stuff in there.

Reply:
Another comment about this. Is there a reason why this .idea folder is visible on the public repository? Seems like something we should add to .gitignore, right?

Reply:
Indeed. What do you think @PaulMcInnis? Should I add this to the .gitignore?

Reply:
Yeah i think it probably shouldn’t have been committed.

Reply:
Will fix asap.